sources/network/sockets/unix-socket-accessor.dylan

```diff
@@ -128,13 +171,6 @@ define function accessor-startup (manager :: <socket-manager>) => (result :: <bo
   socket-manager-started?(manager)
 end function;
 
-// Temporarily disable automatic start-up until maybe we fix dylan
-// initialization so maybe it isn't tied to dll initialization.
-// begin
-//   accessor-startup();
```

Comment:
Is this perhaps worth filing a bug so we deal with it eventually? I'm not familiar with exactly what the issues are. (I do remember when they added start-sockets() though, probably related to this.)

Reply:
I assumed this was related to having to call `WSAStartup` on Windows and therefore not needed for other platforms. I will check, though.